### DISCUSSION TITLE
Pycharm no completions were found, but works well on vscode

### DISCUSSION THREAD

**Discussion by Eilian-Huang** [community/community#7723] [Copilot Conversations] [answered]
Hi, it suddenly doesn't work on my Pycharm. It worked fine last week, but it can't work now.

1. It doesn't work on my Pycharm, but still works fine on my VS Code.
2. No copilot icon shown on my button sidebar in Pychram.
3. I tried open "Tools-GitHub Copilot-Open Github Copilot", it shows No completions were found.

And I have tried:
1. uninstall and install Copilot, and login in
2. uninstall and install Pycharm
3. Reboot my computer

Versions:
PyCharm 2021.2.3 (Professional Edition)
Windows 10 21H1 19043.1348

**Comment by jansorg** [comment]
Could you try `Tools > GitHub Copilot > Login to GitHub`?
This will help to understand what might be wrong.

**Reply by AAdewunmi** [reply]
Thanks @jansorg ... This fix worked for me 👍

**Reply by jiangyong1170** [reply]
I tried many times.  still shows  No completions were found.

**Comment by Eilian-Huang** [comment, marked as answer]
Thanks for your assistance. I tried `Tools > GitHub Copilot > Login to GitHub`, and it told me `Successfully logged in to Github for Github Copilot`.
But I still can't see the icon on the button right. :cry:
It does show one completion in my code, but I cannot get `next completions`, I'm sure it's not about keymap in Pycharm because I have set my own keymap. And it still shows `No completions were found` when I try `Tools > GitHub Copilot > Open GitHub Copilot`

**Reply by jansorg** [reply]
For the IntelliJ plugin, there's no icon in the status bar yet. 
"Previous / next completion" is currently only available for Python and Java files, but the next update will extend this to all file types.
So far, it seems to be working as expected, but most of the limitations should be solved with the next updates.

Regarding "No completions were found", could you post a screenshot of an example where this happened?

**Reply by Eilian-Huang** [reply]
Oh, I have seen the complements! :rofl: Sorry :sob:
But still can't see the icon on the button. Is this a problem or I can just ignore this?

**Reply by jansorg** [reply]
You can ignore the missing icon, it's currently just not implemented.

**Reply by Eilian-Huang** [reply]
Thanks for your help! I think it works as expected. It shows "No completions were found" maybe there it's no code for generating. :no_mouth:
And I had to say it's an amazing plugin and it helps me a lot in coding! :smiley:

**Comment by yashrathi-git** [comment]
I have the same problem, @jansorg solution didn't worked for me. The plugin worked fine before update.

**Reply by yashrathi-git** [reply]
Restarting the IDE fixed it

**Comment by nishikantgurav** [comment]
No completions are showing in Pycharm after update

**Reply by AAdewunmi** [reply]
Hi @nishikantgurav,
Check your Github copilot status to make sure you are logged in.
If you are logged in and the problem persists, toggle the GitHub copilot button (on-off-on), that should make completions work.

<img width="270" alt="Screenshot" src="https://user-images.githubusercontent.com/15172744/174061944-dc32d32a-97b3-4990-b615-066988e7c30b.png">

**Comment by alex-stellar** [comment]
I had the same problem as @Eilian-Huang, I tried to toggle completions as well as log in / log out plugin from GitHub, but it didn't help. Only by hovering over the icon for a while (absolutely accidentally!) I managed to get this error message:
![image](https://github.com/community/community/assets/141749978/7986b22c-ef56-4555-8f42-9825bfe30e95)
Still, marketplace do not provide any newer version of Copilot, so the issue is not fixed

**Reply by alex-stellar** [reply]
Removal of the plug in and installing it again did not help either
